e2e/run_test.go

```diff
@@ -0,0 +1,48 @@
+//
+// This is the e2e package to run tests for Ignite
+// Currently, we do local e2e tests only
+// we have to wait until the CI setup to allow Ignite to run with sudo and in a KVM environment.
+//
+// How to run tests:
+// sudo IGNITE_E2E_HOME=$PWD $(which go) test ./e2e/.
```

Comment:
adding `-count 1` will disable caching
```suggestion
// sudo IGNITE_E2E_HOME=$PWD $(which go) test ./e2e/. -count 1
```